packages/itwinui-react/src/core/Buttons/IconButton.tsx

```diff
@@ -63,7 +63,7 @@ export const IconButton = React.forwardRef((props, ref) => {
   const button = (
     <ButtonBase
       ref={ref}
-      className={cx('iui-button', className)}
+      className={cx('iui-button iui-field', className)}
```

Comment:
This can be written as a list, similar to AvatarGroup.

https://github.com/iTwin/iTwinUI/blob/e2e6ba7abbb46ac99ed225775b56942b88d8da04/packages/itwinui-react/src/core/AvatarGroup/AvatarGroup.tsx#L112-L116

Same comment in other places too.

Reply:
Done.  I caught 1 in toast that looks like it came from someone else's PR, ignored that 1.